dracut/image-boot/eos-image-boot-setup

```diff
@@ -18,12 +18,30 @@ if getargbool 0 endless.live_boot; then
   kpartx_roflag=-r
 fi
 
-fstype=$(lsblk --noheadings -o FSTYPE "${host_device}")
+fstype=$(lsblk -d --noheadings -o FSTYPE "${host_device}")
 if [ $? != 0 ]; then
   echo "image-boot: failed to detect filesystem type"
   exit 1
 fi
 
+if [ "$fstype" = "iso9660" ]; then
+    loopdev="/dev/mapper/endless-image"
```

Comment:
Can we explain what we're doing in this block? “For ISO host filesystems, we can use a normal loop device, but its path needs to match the device created by the dm-setup logic on other filesystems.” perhaps?